android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -604,6 +604,26 @@ class StripeSdkModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
     }
   }
 
+  @ReactMethod
+  fun isGooglePaySupported(params: ReadableMap?, promise: Promise) {
```

Comment:
Is it possible to require an Activity or Fragment at this point? If so, can you use [GooglePayPaymentMethodLauncher](https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt#L113-L154)? After you initialize it it'll make the same request and [call the `readyCallback`](https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt#L203-L208) with the result. That way we don't need to add a dependency to `play-services-wallet` and have a single implementation of this check.

Reply:
Good idea, most recent commits change to using a Fragment w/ `GooglePayPaymentMethodLauncher`

Reply:
Nice, thank you!